browser/src/neovim/NeovimInstance.ts

```diff
@@ -482,27 +497,33 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
             // Workaround for bug in neovim/node-client
             // The 'uiAttach' method overrides the new 'nvim_ui_attach' method
             Performance.startMeasure("NeovimInstance.Start.Attach")
-            return this._attachUI(size.cols, size.rows).then(
-                async () => {
-                    Log.info("Attach success")
-                    Performance.endMeasure("NeovimInstance.Start.Attach")
-
-                    // TODO: #702 - Batch these calls via `nvim_call_atomic`
-                    // Override completeopt so Oni works correctly with external popupmenu
-                    // await this.command("set completeopt=longest,menu")
-
-                    // set title after attaching listeners so we can get the initial title
-                    await this.command("set title")
-
-                    Performance.endMeasure("NeovimInstance.Start")
-
-                    this._initComplete = true
-                },
-                (err: any) => {
-                    this._onError(err)
-                    this._initComplete = true
-                },
-            )
+            return this._attachUI(version, size.cols, size.rows)
+                .then(async () => {
+                    if (startUpErrorsChecked === false) {
+                        this._checkAndFixIfBlocked()
```

Comment:
it seems this call is no longer being `awaited` at @CrossR but the actual function is async could relate to the issues your having

Reply:
Whoops! I remembered enough to make the function async, then forgot it somehow.

I think the issues with the CI is actually because of me changing the notification text, so pushing up a fix for that now.